Hudl.Mjolnir/Command/Attribute/FireAndForgetAttribute.cs

```diff
@@ -13,7 +13,7 @@ namespace Hudl.Mjolnir.Command.Attribute
     /// 
     /// Should only be used on interface methods.
     /// </summary>
-    [Obsolete("Use [FireAndForget] from the Hudl.Mjolnir.Attributes package instead")]
+    [Obsolete("Will be removed in a future release. Leaving async calls un-awaited is generally not recommended.")]
```

Comment:
Doesn't seem to be used a lot? https://github.com/search?q=FireAndForget+user%3Ahudl&ref=searchresults&type=Code&utf8=%E2%9C%93

Worth removing straight away?